dts/arm/st/f4/stm32f411.dtsi

```diff
@@ -27,5 +27,25 @@
 			status = "disabled";
 			label = "SPI_5";
 		};
+
+		i2s4: i2s@40013400 {
+			compatible = "st,stm32-i2s";
+			#address-cells = <1>;
+			#size-cells = <0>;
+			reg = <0x40013400 0x400>;
+			interrupts = <84 5>;
```

Comment:
Miss "clocks"

Reply:
I will add it. But in a separate commit we should probably fix also the spi (clocks is missing also there).
BTW, I wonder how the clocks are currently provided to driver. It seems that currently this 'clocks' declaration in dts is not used yet.

Reply:
Ooooh! OK, got it.
I should add clock there, which means the address + bit at which the clock gating is placed.
I can do that, but then I would also need to fix the driver, as currently I'm configuring the clock gating there, but instead I will obtain it from DTS.

For example now I have in I2S driver:
   806 >       .pclken = {
   807 >       >       .enr = LL_APB2_GRP1_PERIPH_SPI1,
   808 >       >       .bus = STM32_CLOCK_BUS_APB2,
   809 >       },

This part I guess will disappear, as now it will be in DTS.

Reply:
I'll do some tests!

Reply:
Yes, indeed, you'll get these from dts as well

Reply:
done for all!

Reply:
Well... they should. Did you had troubles when using thoses?

Reply:
By the way, you should use the CONFIG_ equivalenst coming from fixup files, not the ST_STM32_...

Reply:
So, @avisconti ,  I just created https://github.com/zephyrproject-rtos/zephyr/issues/10435 to get this done one I2C and will do the same for other drivers that don't do it yet.
Would be nice you can do it right now on I2S.

Reply:
@erwango 
OK, agree.  I'm taking a look at it right now.

Reply:
> By the way, you should use the CONFIG_ equivalenst coming from fixup files, 
> not the ST_STM32_...

Yes, yes, of course. I already investigated it yesterday. But I stopped it because it was not done for any IP (I2c, SPI, ...).